README.md

```diff
@@ -1,2 +1,87 @@
+- The Development Coordination Team.
+- A number of smaller _project groups_.
+
+## The Development Coordination team
+
+The Development Coordination team (a.k.a. the _Dev Team_) is a group of volunteers that are chosen annually in a similar fashion to the formation of the _Core Team_, although they may be responsible for projects beyond the annual timescale.
+
+The goals of the _Dev Team_ are to ensure the long-term longevity of Student Robotics by working on projects that enhance the experience for SR’s competitors and volunteers in line with its values, and enable SR’s long term growth. Additionally they will aim to fulfill the values of Student Robotics, in particular our goal of reflecting reality by incorporating the latest technology whenever feasible.
+
+## Project Groups
+
+A _project group_ is a smaller group of self-selected volunteers who have a particular interest in working on something together. The _project group_ should report progress regularly to the _Dev Team_. The _Dev Team_ oversees the various _project groups_ to ensure that they are on track and aligned with the goals of the charity. _Project groups_ are responsible for maintaining projects that they have.
+
+### Formation of a Project Group
+
+A group of volunteers decides that they would like to work on a project. They communicate their ideas to the _Dev Team_ and the _Dev Team_ will decide whether the project is beneficial to the organisation. A schedule for the project will be discussed between the _Dev Team_ and the _project group_. A _project group_ may apply to the _Dev Team_ for budget for specific purchases for their project, given sufficient justification. The _Dev Team_ should also keep in mind the future needs of _project groups_ when planning their budget.
+
+Alternatively, the _Dev Team_ may identify the need for a project within the organisation and will find interested volunteers to form a group to do the project.
+
+## Budgeting
+
+The _Dev Team_ is allocated a budget, to be agreed with the Trustees. The scope of this budget is to cover costs incurred for development activities only, and not the cost of deploying any completed projects which would be arranged between the trustees and _Core Team_.
+
+For the purposes of budgetary accountability, the _Dev Team_ will select a Chair and Treasurer. (How the budget is managed is between the _Dev Team_ and the Trustees.) In practice, unlike the _Core Team_ budget which is fairly detailed (e.g. £x on Kickstart, £y on Competition, £z on other) and is approved in advance, _Dev Team_ purchasing will be defined at a higher level (maybe some “infrastructure” budget for a dev server and some dev tools but not £a for Project A, £b for Project B. Unlike the _Core Team_, this will not be a rolling budget, but a “pot” that gets topped up in each accounting period to allow for projects to be budgeted over multiple years.
```

Comment:
@RealOrangeOne the problem with a non rolling pot is that funding for multi year projects cant be reserverd for the lifetime of the project. e.g. development of a new board that will take more than a year to complete and will have costs throughout the project.

Reply:
Maybe project proposals should include a planned budget for the lifetime of the project. If the project proposal is accepted, funding for future years will be set aside in accordance with this budget. This would help avoid the situation where e.g. development of a new board has to pause because there is not enough money left in the current year's pot to fund manufacture.

Reply:
I agree with @kierdavis that having something in the proposal about the expected budget is a good idea. 

I think we should defer to the trustees the actual implementation of how the budget works.

Reply:
> I think we should defer to the trustees the actual implementation of how the budget works.

Agreed. I think the core team should make a suggestion to them though.

Reply:
Agreeed. I think what's written here is fine to cover that.